development/pubsub-connector/pubSubGateway/README.md

```diff
@@ -0,0 +1,31 @@
+# PubSub Gateway
+
+## Overview
+
+The PubSub Gateway is written in Golang. It pulls messages from PubSub, converts them into a CloudEvents compliant events and forwards them to the Kyma Event Publisher Proxy.
```

Comment:
```suggestion
The PubSub Gateway is written in Golang. It pulls messages from PubSub, converts them into Cloud Events, and forwards them to the Kyma Event Publisher Proxy.
```

Reply:
CloudEvents is a name of specification. I think it should not be written in two separate words. Check their page. https://cloudevents.io/